# Project: Crisis in Beaufort (Apply Statistics)
# Name:       Asim Jana
# E-mail: asim.jana@gmail.com

Problem Statement:
In Beaufort Police Department, Chief Police Officer called great Data Scientist, Patrick & told that " I want you to look at the data and come up with a proposal to tackle crime in the city".

Now, Patrick has following steps:
1. Identify the different ‘Crime’categories.
2. Gather data for the past 10 years.
3. Analyze the data for any trends.
4. Report his findings to Jemes.

There are different crimes like Car theft, Burglary, Bike theft, Violence, Vandalism.
Patrick collected below data for Car Theft incidents.
1. Total Car Theft incidents in the past decade: 1500

Car_Thefts_yr Count
2008	  151
2009	  158
2010	  161
2011	  148
2012	  155
2013	  194
2014	  140
2015	  169
2016	  172

This is data set for year 2017.
year: 2017
-----------
Month	Count
Jan	     8
Feb	    14
Mar	    21
Apr	    25
May	    33
Jun	    ?

What should be the count for jun, 2017? 
What should be the next actions & solutions from Patrick end?

1. To identify trends, need to summarize the data and need to calculate mean.

In [12]:
import numpy as np
import pandas as pd

In [21]:
# dictionary data
Dataf = {'Car_Thefts_yr': [2008,2009,2010,2011,2012,2013,2014,2015,2016], 'Count_yr': [151,158,161,148,155,194,140,169,172]}

#assigning data into pandas dataframe
Datac = pd.DataFrame(data=Dataf)
Datac

Car_Thefts_yr  Count_yr
0           2008       151
1           2009       158
2           2010       161
3           2011       148
4           2012       155
5           2013       194
6           2014       140
7           2015       169
8           2016       172

In [23]:
#After looking into dataset, I should calculate MEAN to find out the count for the year 2017
Mean2017 = np.mean(Datac['Count_yr'])
Mean2017

160.88888888888889

In [28]:
#Standard deviation of above data set.
std_dev_2017 = np.std(Datac['Count_yr'])
std_dev_2017

15.014396383652327

In [29]:
percentage_of_std_dev_2017 = (std_dev_2017/Mean2017)*100
percentage_of_std_dev_2017

9.33215244840269

# Standard deviation is less than 10%. 
1. In a year on average 161 car theft and there my be additional 15 cars stolen or 15 cars not stolen.
2. Between 146 to 176 cars are being stolen in every year.
3. With above knowledge, I should examine what is happening over time to see if any increases had appeared.

So, need to analyze monthly data for 2017.
This would tell what was happening this year, by month, and point out any increases that appear to be related to a crime pattern. 

In [17]:
#Data set for year 2017
Dataf2017 = {'Month_2017': ['Jan','Feb','Mar','Apr','May'], 'Count_2017': [8,14,21,25,33]}
Datac2017 = pd.DataFrame(data = Dataf2017)
Datac2017

Count_2017 Month_2017
0           8        Jan
1          14        Feb
2          21        Mar
3          25        Apr
4          33        May

In [32]:
Mean2017mon = np.mean(Datac2017['Count_2017'])
Mean2017mon

20.2

In [33]:
Std_dev = np.std(Datac2017['Count_2017'])
Std_dev

8.657944328765346

In [34]:
percentage_of_Std_dev = (Std_dev/Mean2017mon)*100
percentage_of_Std_dev

42.86111053844231

In [39]:
total_theft_2017 = (Mean2017mon *12)
remain_total_theft_2017 = (Mean2017mon *12) - np.sum(Datac2017['Count_2017'])
print(total_theft_2017)
print(remain_total_theft_2017)
avg_of_rest_mon2017 = (remain_total_theft_2017/7)
avg_of_rest_mon2017

242.39999999999998
141.39999999999998


20.199999999999996

Standard deviation is 43% (high value) and the average monthly value is 20.
1. Upon examine of the monthly report, it is found that there did appeared to be a rise in the number of cars stolen over the past five month.
2. But average number of cars stolen this year for eavery month is 20.

# Report Findings:


1. It is found that the Cars Thefta are indeed increasing quite alarmingly compared to the average (242 vs 161).
2. Based on the above findings, James should deploy specialized police investigators trained in handling car burglaries 
across Beaufort